lib/lita/handlers/trello.rb

```diff
@@ -7,90 +5,110 @@ class Trello < Handler
       config :token
       config :board
 
-      route(/^trello\s+new\s+[^\s]+\s+[^\s]+/i, :handle_create_card, command: true, help: {
-        "trello new LIST Do the thing" => "Create a new card in LIST called 'Do the thing'"
-      })
-      def handle_create_card(r)
-        list_name = r.args[1]
-        name = r.args[2..-1].join(" ")
+      route(
+        /^trello\s+new\s+(?<list_name>\w+)\s+(?<name>.*)/i,
```

Comment:
The named capture groups are much more readable :+1:

Sadly `\w+` breaks matching list names that contain spaces; `Lita::Message#args` does shell-like splitting, allowing you to e.g. `lita trello new "to do" foo` to add a card to the "to do" board. Maybe `(?<list_name>(\w+|"[^"]+"))`?